crates/aptos/src/genesis/config.rs

```diff
@@ -6,20 +6,27 @@ use crate::{
     genesis::git::from_yaml,
 };
 use aptos_crypto::{ed25519::Ed25519PublicKey, x25519};
-use aptos_types::network_address::DnsName;
+use aptos_types::{chain_id::ChainId, network_address::DnsName};
 use serde::{Deserialize, Serialize};
-use std::{fs::File, io::Read, path::PathBuf};
+use std::{fs::File, io::Read, path::PathBuf, str::FromStr};
 
 /// Template for setting up Github for Genesis
 ///
-/// Previously known as layout
 #[derive(Debug, Deserialize, Serialize)]
-pub struct Participants {
+pub struct Layout {
+    /// Root key for the blockchain
+    /// TODO: In the future, we won't need a root key
+    pub root_key: Ed25519PublicKey,
+    /// List of usernames or identifiers
     pub users: Vec<String>,
+    /// ChainId for the target network
+    pub chain_id: ChainId,
+    /// Modules folder
+    pub modules_folder: String,
```

Comment:
I kinda like enforcing that this is in a specific location :P

Reply:
also if you add this, it should be PathBuf, no?

Reply:
It's not really pathbuf, because GitHub paths end up being strings.

We can just make this a specific location